sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/Generated/CassandraClusterData.Serialization.cs

```diff
@@ -47,7 +47,7 @@ protected override void JsonModelWriteCore(Utf8JsonWriter writer, ModelReaderWri
             if (Optional.IsDefined(Identity))
             {
                 writer.WritePropertyName("identity"u8);
-                JsonSerializer.Serialize(writer, Identity);
+                JsonSerializer.Serialize(writer, Identity, AzureResourceManagerCosmosDBJsonContext.Default.ManagedServiceIdentity);
```

Comment:
We should modify this to pass in a common instance of JsonSerializerOptions which uses https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/System.ClientModel/src/ModelReaderWriter/JsonModelConverter.cs and specifically calls the ctor that takes in `AzureResourceManagerCostmosDBContext.Default`

```c#
//internal helper somewhere
    internal static JsonSerializerOptions s_options = new()
    {
        Converters =
        {
            new JsonModelConverter(ModelSerializationExtensions.WireOptions, AzureResourceManagerCostmosDBContext.Default)
        }
    };
```

```c#
//this line then becomes
JsonSerializer.Serialize(writer, Identity, s_options);
```